packages/custom-elements/externs/custom-elements.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @externs
+ * @license
+ * Copyright (c) 2021 The Polymer Project Authors. All rights reserved. This
+ * code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+// This file needs to be a module for the declarations to apply globally.
```

Comment:
here and elsewhere: kinda the opposite is the case. declarations in non-modules are global by default

but for consistency and our own sanity, we prefer all files to be modules

Reply:
Yeah, this comment doesn't make sense; updated.